e2e/pages/adf/process-cloud/processListCloudComponent.ts

```diff
@@ -44,4 +59,73 @@ export class ProcessListCloudComponent {
         return this.dataTable.getAllRowsColumnValues(column);
     }
 
+    getNoOfCoulmns() {
```

Comment:
All methods related to datatable (rows, columns, etc) should be moved inside datatable PO. I think there is already a method checkColumIsDisplayed that can be used.

Reply:
Added the method required and calling it from DataTable.